modin/pandas/groupby.py

```diff
@@ -116,24 +116,33 @@ def __getattr__(self, key):
     def ngroups(self):
         return len(self)
 
-    def skew(self, **kwargs):
-        return self._apply_agg_function(lambda df: df.skew(**kwargs))
+    def skew(self, *args, **kwargs):
+        return self._wrap_aggregation(
+            type(self._query_compiler).groupby_skew,
```

Comment:
Are these all just defaulting to pandas now?

Reply:
no, all of them do like [`query_compiler.groupby_skew()`](https://github.com/dchigarev/modin/blob/98f54ae176b89556937768cdaafee8ebc06862ab/modin/core/storage_formats/base/query_compiler.py#L2417) -> [`query_compiler.groupby_agg(agg_func="skew")`](https://github.com/dchigarev/modin/blob/98f54ae176b89556937768cdaafee8ebc06862ab/modin/core/storage_formats/base/query_compiler.py#L2428), it's then `storage_format_qc.groupby_agg` responsobility of how to process the function (default to pandas or not), for example, at pandas storage format, `pandas_qc.groupby_agg` always [executes groupby as a full-axis function](https://github.com/modin-project/modin/blob/0faf4675140415e17d4112f9d0d37cfe87770b9e/modin/core/storage_formats/pandas/query_compiler.py#L2785-L2786), so there's no fallback to pandas.